### PR TITLE
1 blog listing (4)[2]

### PR DESCRIPTION
**Pull request by rcaracaus**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -7,9 +7,13 @@ app.engine('hbs', hbs.engine);
 
 app.set('view engine', 'hbs');
 
+app.set('views', 'home');
```
Comment (rcaracaus) [inline app.js]: check if we need this

**Comment by edisch** [comment]
@JordanDDisch Considering writing a service for listings.

File: lib/blog-listing/blog-listing.hbs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{{#each resource }}
+	<div class="listing">
+	    <img class="image-float" src="{{ field_image.url }}" />
+		<h3 class="heading--third">
```
Comment (rcaracaus) [inline lib/blog-listing/blog-listing.hbs]: should use h2

File: lib/blog-listing/index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+		return Promise.settle(PromiseArray);
+
+	}).then(function (view) {
+
+		view.resource = [];
+
+		view.forEach(function (element, index) {
+			view.resource[index] = element._settledValue;
+			view.resource[index].url = removeBaseUrl(view.resource[index].url);
+			view.resource[index].body.value = truncate(view.resource[index].body.value, 200, "...");
```
Comment (rcaracaus) [inline lib/blog-listing/index.js]: strip tags?

**Pull request merged**